nf/bpf/bridge/include/uapi/linux/bpf.h

```diff
@@ -50,18 +50,19 @@
 #define BPF_CMPXCHG (0xf0 | BPF_FETCH) /* atomic compare-and-write */
```

Comment:
this file was taken directly from some Linux source, it'd be good to not modify it at all

Reply:
reverted and added an exclude regex to the `lint-env-and-nf` job